### PR TITLE
Typo fixed and Spanish (Spain) translations

### PR DESCRIPTION
**Pull request by xusy2k**
Typo fixed "ambiguious" in code "invalid_bidirectional" at validators.py
Updated Spanish (Spain) translations

### CODE REVIEW

**Comment by tarak** [comment]
Hey, thank you very much for this PR. I definitely woiuld like to merge it, but you have changed several strings that are to be translated. To merge would mean all the other translation files would be broken. I need to review this when I have time, probably during the weekend.

**Comment by xusy2k** [comment]
OK! I only executed django-admin.py makemessages -l es, translated them and compiled

File: password_policies/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -18,83 +18,151 @@ msgstr ""
...
+msgid "Username"
+msgstr "Nombre de usuario"
+
+#: forms/__init__.py:272
+msgid "Required. 30 characters or fewer. Letters, digits and @/./+/-/_ only."
 msgstr ""
+"Obligatorio. 30 caracteres o menos. Sólo letras, dígitos y @ / / + / -. / _"
 
-#: forms/validators.py:152
-msgid "The new password contains ambiguious bidirectional characters."
```
Comment (tarak) [inline password_policies/locale/es/LC_MESSAGES/django.po]: This breaks other translations...

File: password_policies/forms/validators.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ class BidirectionalValidator(BaseRFC4013Validator):
     #: The validator's error code.
     code = u"invalid_bidirectional"
     #: The validator's error message.
-    message = _("The new password contains ambiguious bidirectional characters.")
+    message = _("The new password contains ambiguous bidirectional characters.")
```
Comment (tarak) [inline password_policies/forms/validators.py]: This needs to be updated in other translations as well... Find & replace would be enough I guess.

**Comment by xusy2k** [comment]
Yes, but there are new  translation strings.

BTW: I've released new version with Catalan translations

**Comment by xusy2k** [comment]
I think never remove entries at PasswordChangeRequired, so if you force an unser to change his/her password, always ask to change his/her password

**Comment by tarak** [comment]
Hey, I have reviewed your PR. I would like to ask you a few things:

- Can you please pull upstream and merge it again? I have pulled other PR's...
- Please do not change the version number. If you resubmit a PR I will take of that myself and also upload the new version to PyPI.
- Can you split up the change in xusy2k/django-password-policies@f386ff57f5b0d06b84d9dadadfa8f5447485083b in another PR, please? This is somehow important to me because the PasswordChange needs more work...
- I have changed the typo "ambiguous" in all other django.po files so this has been taken care of. (See https://github.com/tarak/django-password-policies/pull/30#discussion-diff-26559064)

If it's more convenient we could also close this PR and take care of everything in new PR's.

Thank you in advance for your support!

**Pull request merged**